pkg/cmd/spks.go

```diff
@@ -50,14 +50,23 @@ func SpksCMD() *cli.Command {
 			&cli.StringFlag{Name: "sales-order", Usage: "Sales order to report billing data to",
 				EnvVars: []string{"SALES_ORDER"}, Destination: &salesOrder, Required: false, DefaultText: defaultTextForOptionalFlags, Value: "S10121"},
 			&cli.StringFlag{Name: "prometheus-url", Usage: "URL of the Prometheus API",
-				EnvVars: []string{"PROMETHEUS_URL"}, Destination: &prometheusURL, Required: true, DefaultText: defaultTextForRequiredFlags, Value: "http://prometheus-monitoring-application.monitoring-application.svc.cluster.local:9090"},
+				EnvVars: []string{"PROMETHEUS_URL"}, Destination: &prometheusURL, Required: false, DefaultText: defaultTextForRequiredFlags, Value: "http://prometheus-monitoring-application.monitoring-application.svc.cluster.local:9090"},
 			&cli.StringFlag{Name: "unit-id", Usage: "Metered Billing UoM ID for the consumed units",
-				EnvVars: []string{"UNIT_ID"}, Destination: &UnitID, Required: true, DefaultText: defaultTextForRequiredFlags, Value: "uom_uom_68_b1811ca1"},
+				EnvVars: []string{"UNIT_ID"}, Destination: &UnitID, Required: false, DefaultText: defaultTextForRequiredFlags, Value: "uom_uom_68_b1811ca1"},
 		},
 		Action: func(c *cli.Context) error {
 			logger := log.Logger(c.Context)
 			logger.Info("starting spks data collector")
 
+			location, err := time.LoadLocation("Europe/Zurich")
+			if err != nil {
+				return fmt.Errorf("load location: %w", err)
+			}
+			// this variable is necessary to query Prometheus, with timerange [1d:1d] it returns data from 1 day up to midnight
+			startOfToday := time.Date(time.Now().Year(), time.Now().Month(), time.Now().Day(), 0, 0, 0, 0, location)
+			startYesterdayAbsolute := time.Date(time.Now().Year(), time.Now().Month(), time.Now().Day(), 0, 0, 0, 0, location).Add(-(time.Hour * 24)).UTC()
```

Comment:
This is simpler and closer to what we do for Exoscale and cloudscale:
```suggestion
			startYesterdayAbsolute := time.Date(time.Now().Year(), time.Now().Month(), time.Now().Day()-1, 0, 0, 0, 0, location).In(time.UTC)
```